service/src/main/java/org/cbioportal/genome_nexus/service/internal/HotspotFilter.java

```diff
@@ -15,6 +15,9 @@
 @Component
 public class HotspotFilter
 {
+    public static Double MISSENSE_FRACTION_THRESHOLD = 0.25;
+    public static Integer MISSENSE_COUNT_THRESHOLD = 1;
```

Comment:
@inodb should we make this configurable via `application.properties`?

Reply:
Makes sense if in Genome Nexus we are only interested in the subset of hotspots mutations where `missense_fraction >= 25% && missense_count > 1`. I am fine with that solution if it is not a problem to not import all the hotspots mutations we have in Cancer Hotspots website.